sentry/templates/deployment-snuba-subscription-consumer-events.yaml

```diff
@@ -66,18 +66,17 @@ spec:
         imagePullPolicy: {{ default "IfNotPresent" .Values.images.snuba.pullPolicy }}
         command:
           - "snuba"
-          - "subscriptions"
+          - "subscriptions-scheduler-executor"
           - "--partitions={{ .Values.snuba.subscriptionConsumerEvents.partitions}}"
```

Comment:
It seems this isn't needed with `subscriptions-scheduler-executor`, at least this option isn't available anymore, see https://github.com/getsentry/snuba/blob/22.6.0/snuba/cli/subscriptions_scheduler_executor.py vs https://github.com/getsentry/snuba/blob/22.6.0/snuba/cli/subscriptions.py.

Might be worth removing it in next release.